theme/modules/Home_Social.php

```diff
@@ -0,0 +1,83 @@
+<?php
```

Comment:
We should agree on an emplacement for widgets, not the same was chosen here: https://github.com/nuitdebout/wordpress/commit/0860a14f1af33b6d386216d9813ee2e9c7c12c9f#diff-9fd0761fb71e1e6af12afd730b3c7833R97